frontend/src/stories/PanelStack/PanelStack.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from "prop-types";
+import * as BPCore from "@blueprintjs/core";
+
+export function Panel({content}) {
+	return content
+}
```

Comment:
Huh!  I don't know that I have a comment, I'm just curious: is that idiomatic?

Reply:
Technically I am extracting content from the props, but I've now simplified it to an arrow function in the PanelStack, which I hadn't thought to do initially.